build/.vsts-ci.yml

```diff
@@ -264,6 +264,41 @@ phases:
   - script: sudo rm -rf $(Agent.BuildDirectory)/*
     displayName: 'cleanup'
     condition: always()    
+- phase: linux_install_deps
+  variables:
+    _PREVIEW_VSTS_DOCKER_IMAGE: "aziotbld/c-debian"
+  queue: aziotbld-lin01
+  displayName: 'linux_install_deps'
+  steps:
+  - script: |
+     if [ -f "jenkins/linux_install_deps.sh" ]
+     then
+      sudo chmod 755 jenkins/linux_install_deps.sh
+      sudo -E ./jenkins/linux_install_deps.sh
+      sudo jenkins/linux_install_deps.sh
+     fi
+    displayName: 'build'
+    env:
```

Comment:
Do we need these environment variables if we are not running the test?

Reply:
Probably not. Shall remove them in another PR so we can scan the whole yaml file for jobs to clean